src/executor/attach_op_execs_pass.cc

```diff
@@ -62,6 +71,13 @@ class BackwardOpExecutor : public OpExecutor {
     op_ctx.run_ctx = rctx;
     op_->Backward(op_ctx, out_grad_, in_data_, out_data_,
                   req, in_grad_, aux_data_);
+#if MKL_EXPERIMENTAL == 1
+    mkl_tblobs_prv_to_cpu(out_grad_);
+    mkl_tblobs_prv_to_cpu(in_data_);
+    mkl_tblobs_prv_to_cpu(out_data_);
```

Comment:
without patch: INFO:root:Epoch[0] Batch [390]  Speed: 178.12 samples/sec       Train-accuracy=0.132812 for resnet

with patch INFO:root:Epoch[0] Batch [390]  Speed: 199.15 samples/sec       Train-accuracy=0.367188